package.json

```diff
@@ -61,7 +61,8 @@
     "nyc": "^15.0.0",
     "publish-please": "^5.4.3",
     "sinon": "^1.17.4",
-    "sinon-chai": "^2.8.0"
+    "sinon-chai": "^2.8.0",
+    "babel-cli":  "^6.26.0"
```

Comment:
Do we need babel-core too and do we need to update the other babel dependencies to match this one?

Reply:
as per the nodejs support team they asked us to use only "babel-cli":  "^6.26.0" and raise a new PR with yml file. Mostly this should fix the issue. if the issue still exist then i will update the other babel as well.